CoreFoundation/Base.subproj/CoreFoundation.h

```diff
@@ -33,6 +35,7 @@
 #include <stdio.h>
 #include <stdlib.h>
 #include <string.h>
+#include <netdb.h>
```

Comment:
These are a global change to both the swift-corelibs, CFLite and Darwin versions, changes to these types of headers come at definite risk. Can this be done without these modifications in CoreFoundation.h? (and perhaps somewhere else instead?)